src/server/game/Server/Packets/SystemPackets.cpp

```diff
@@ -121,6 +121,8 @@ WorldPacket const* FeatureSystemStatus::Write()
     _worldPacket.WriteBit(ChatDisabledByDefault);
     _worldPacket.WriteBit(ChatDisabledByPlayer);
     _worldPacket.WriteBit(LFGListCustomRequiresAuthenticator);
+    _worldPacket.WriteBit(RaceClassExpansionLevels.has_value());
+    _worldPacket.WriteBit(BattlegroundsEnabled);
```

Comment:
I believe this bit only enable/disable War Games and not battlegrounds entirely

Reply:
> I believe this bit only enable/disable War Games and not battlegrounds entirely

```cpp
  if ( !byte_143621604 )
  {
    CGGameUI::DisplayError(1039);
    return 0i64;
  }
```
This is the check that checks the bool value and the value 1039 corresponds to `ERR_BATTLEGROUND_NOT_ENABLED`

Reply:
Update: Indeed related to wargame, blame blizzard for not naming the error code correctly.